Examples/ArcGISToolkitExamples/JobManagerExample.swift

```diff
@@ -326,13 +350,17 @@ class JobManagerExample: TableViewController {
                 
                 // register the job with our JobManager shared instance
                 JobManager.shared.register(job: job)
+
+                // try to keep the app running in the background for this job if possible
+                let backgroundTaskIdentifier = self.startBackgroundTask()
```

Comment:
same as above, consider adding this at the beginning of `takeOffline`. Apple's doc suggests calling this "as early as possible before starting your task"

> Call this method as early as possible before starting your task, and preferably before your app actually enters the background. The method requests the task assertion for your app asynchronously. If you call this method shortly before your app is due to be suspended, there is a chance that the system might suspend your app before that task assertion is granted.